lang/CMakeLists.txt

```diff
@@ -71,6 +71,7 @@ set(sclang_sources
 	LangSource/SimpleStack.cpp
 	LangSource/VMGlobals.cpp
 	LangSource/dumpByteCodes.cpp
+        LangSource/pyslottest.cpp
```

Comment:
Indent incorrect?

Reply:
This file needs removing anyway. I haven't looked at how to add cpp tests yet.